org.kde.labplot.json

```diff
@@ -0,0 +1,437 @@
+{
+    "app-id": "org.kde.labplot",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.9",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.9",
+    "sdk": "org.kde.Sdk",
+    "command": "labplot",
+    "rename-icon": "labplot",
+    "finish-args": [
+        "--filesystem=host",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/cmake",
+        "/lib/pkgconfig",
+        "/lib/*.la",
+        "/lib/*.a",
+        "/mkspecs",
+        "/share/aclocal",
+        "/share/info",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "gsl",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz",
```

Comment:
you should find a mirror for this, gnu.org is unreliable inside github actions runners

Reply:
changed to ftpmirror.gnu.org